jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/dto/GroupAdministrationDto.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (C) 2011  JTalks.org Team
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+package org.jtalks.jcommune.web.dto;
+
+import org.jtalks.common.model.entity.Group;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+/**
+ * DTO class for user group containing ID,name and number of users in the group*
+ *
+ * @author Oleg Tkachenko
+ */
+
+public class GroupAdministrationDto {
+
+    private long id;
+    private String name;
+    private long numberOfUsers;
+
+    public GroupAdministrationDto(){}
+    /**
+     * Instantiates new GroupDto object based on the Group object
+     *
+     * @param group source data for the DTO object
+     */
+    public GroupAdministrationDto(Group group) {
+        if (group != null) {
+            this.id = group.getId();
+            this.name = group.getName();
+            this.numberOfUsers = group.getUsers().size();
```

Comment:
That would require to select all the users in the list unless the lazy=extra. We agreed that we don't want to select all the users, right?